### ISSUE TITLE
Gitter chat seems dead

### ISSUE THREAD

**Issue by adrian5** [nvim-telescope/telescope.nvim#2300]
The GitHub Issue → Question template leads to the Gitter chat, which is also mentioned in the project README. Going there I mostly see unanswered questions. I'm wondering if there's a better solution than users sending their questions into the void, while not burdening developers too much.

**Comment by Conni2461** [comment]
Yeah i am sorry, I currently dont have time to check gitter/matrix, so i am thinking that i just remove the mention and do a github question issue template!? That way i only have to check one place.

Alternative is doing a discord server but that might just end up resulting in the same issue

**Comment by gegoune** [comment]
How about enabling discussions rather than adding question template?

**Comment by Conni2461** [comment]
We tried them and i didn't like them. Its harder for me to understand whats done and what still needs an answer.

**Comment by clason** [comment]
Discussions have been enabled for a while.
